packages/inter-protocol/src/proposals/update-vf-electorate.js

```diff
@@ -0,0 +1,35 @@
+import { makeTracer } from '@agoric/internal';
+import { E } from '@endo/far';
+
+const trace = makeTracer('ReplaceFeeDistributer', true);
```

Comment:
We can just run this as a core eval script if we avoid `import` / `export`.

Drop it into [CoreEval form on cosgov for devnet](https://cosgov.org/agoric?msgType=coreEvalProposal&network=devnet) with a `permit.json` containing `true`.

```suggestion
const trace = (...args) => console.log('ReplaceFeeDistributer', ...args);
```

Reply:
I suppose the permit should be `JSON.stringify({
      consume: {
        economicCommitteeCreatorFacet: true,
        vaultFactoryKit: true,
      },
    })`